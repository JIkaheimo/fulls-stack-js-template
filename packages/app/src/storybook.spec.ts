import initStoryshots, { snapshotWithOptions } from '@storybook/addon-storyshots';

const createNodeMock = () => document.createElement('div');

initStoryshots({
  framework: 'vue3',
  test: snapshotWithOptions(createNodeMock),
});
