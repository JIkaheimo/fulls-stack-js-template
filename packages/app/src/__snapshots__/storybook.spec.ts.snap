// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Base/Button Primary 1`] = `
<button
  class="btn"
>
  asd
</button>
`;

exports[`Storyshots Tasks/PureTaskList Default 1`] = `
<button
  class="btn"
>
  asd
</button>
`;

exports[`Storyshots Tasks/PureTaskList Empty 1`] = `
<div
  style="margin: 3em;"
>
  <div
    class="shadow-md w-100"
  >
    
    <div
      class="shadow-sm p-3"
    >
      <svg
        height="40"
        preserveAspectRatio="xMidYMid meet"
        version="1.1"
        viewBox="0 0 100% 40"
        width="100%"
      >
        <rect
          clip-path="url(#l9f9ouyjjup)"
          height="100%"
          style="fill: url(#d7ctpnxsm7t);"
          width="100%"
          x="0"
          y="0"
        />
        <defs>
          <clippath
            id="l9f9ouyjjup"
          >
            
            <circle
              cx="15"
              cy="20"
              r="15"
            />
            <rect
              height="40"
              width="100%"
              x="45"
              y="5"
            />
            
          </clippath>
          <lineargradient
            id="d7ctpnxsm7t"
          >
            <stop
              offset="0%"
              stop-color="#f3f3f3"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-2; 1"
              />
            </stop>
            <stop
              offset="50%"
              stop-color="#ecebeb"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-1.5; 1.5"
              />
            </stop>
            <stop
              offset="100%"
              stop-color="#f3f3f3"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-1; 2"
              />
            </stop>
          </lineargradient>
        </defs>
      </svg>
    </div>
    <div
      class="shadow-sm p-3"
    >
      <svg
        height="40"
        preserveAspectRatio="xMidYMid meet"
        version="1.1"
        viewBox="0 0 100% 40"
        width="100%"
      >
        <rect
          clip-path="url(#tyazlnpayqj)"
          height="100%"
          style="fill: url(#aqv6zyr1bwh);"
          width="100%"
          x="0"
          y="0"
        />
        <defs>
          <clippath
            id="tyazlnpayqj"
          >
            
            <circle
              cx="15"
              cy="20"
              r="15"
            />
            <rect
              height="40"
              width="100%"
              x="45"
              y="5"
            />
            
          </clippath>
          <lineargradient
            id="aqv6zyr1bwh"
          >
            <stop
              offset="0%"
              stop-color="#f3f3f3"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-2; 1"
              />
            </stop>
            <stop
              offset="50%"
              stop-color="#ecebeb"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-1.5; 1.5"
              />
            </stop>
            <stop
              offset="100%"
              stop-color="#f3f3f3"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-1; 2"
              />
            </stop>
          </lineargradient>
        </defs>
      </svg>
    </div>
    <div
      class="shadow-sm p-3"
    >
      <svg
        height="40"
        preserveAspectRatio="xMidYMid meet"
        version="1.1"
        viewBox="0 0 100% 40"
        width="100%"
      >
        <rect
          clip-path="url(#urlzj0bewu)"
          height="100%"
          style="fill: url(#jq85dh5ut0i);"
          width="100%"
          x="0"
          y="0"
        />
        <defs>
          <clippath
            id="urlzj0bewu"
          >
            
            <circle
              cx="15"
              cy="20"
              r="15"
            />
            <rect
              height="40"
              width="100%"
              x="45"
              y="5"
            />
            
          </clippath>
          <lineargradient
            id="jq85dh5ut0i"
          >
            <stop
              offset="0%"
              stop-color="#f3f3f3"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-2; 1"
              />
            </stop>
            <stop
              offset="50%"
              stop-color="#ecebeb"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-1.5; 1.5"
              />
            </stop>
            <stop
              offset="100%"
              stop-color="#f3f3f3"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-1; 2"
              />
            </stop>
          </lineargradient>
        </defs>
      </svg>
    </div>
    <div
      class="shadow-sm p-3"
    >
      <svg
        height="40"
        preserveAspectRatio="xMidYMid meet"
        version="1.1"
        viewBox="0 0 100% 40"
        width="100%"
      >
        <rect
          clip-path="url(#21dyiklfzha)"
          height="100%"
          style="fill: url(#vtes7xs1zwr);"
          width="100%"
          x="0"
          y="0"
        />
        <defs>
          <clippath
            id="21dyiklfzha"
          >
            
            <circle
              cx="15"
              cy="20"
              r="15"
            />
            <rect
              height="40"
              width="100%"
              x="45"
              y="5"
            />
            
          </clippath>
          <lineargradient
            id="vtes7xs1zwr"
          >
            <stop
              offset="0%"
              stop-color="#f3f3f3"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-2; 1"
              />
            </stop>
            <stop
              offset="50%"
              stop-color="#ecebeb"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-1.5; 1.5"
              />
            </stop>
            <stop
              offset="100%"
              stop-color="#f3f3f3"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-1; 2"
              />
            </stop>
          </lineargradient>
        </defs>
      </svg>
    </div>
    <div
      class="shadow-sm p-3"
    >
      <svg
        height="40"
        preserveAspectRatio="xMidYMid meet"
        version="1.1"
        viewBox="0 0 100% 40"
        width="100%"
      >
        <rect
          clip-path="url(#tbuokpcvyy)"
          height="100%"
          style="fill: url(#uljfldqw08a);"
          width="100%"
          x="0"
          y="0"
        />
        <defs>
          <clippath
            id="tbuokpcvyy"
          >
            
            <circle
              cx="15"
              cy="20"
              r="15"
            />
            <rect
              height="40"
              width="100%"
              x="45"
              y="5"
            />
            
          </clippath>
          <lineargradient
            id="uljfldqw08a"
          >
            <stop
              offset="0%"
              stop-color="#f3f3f3"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-2; 1"
              />
            </stop>
            <stop
              offset="50%"
              stop-color="#ecebeb"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-1.5; 1.5"
              />
            </stop>
            <stop
              offset="100%"
              stop-color="#f3f3f3"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-1; 2"
              />
            </stop>
          </lineargradient>
        </defs>
      </svg>
    </div>
    <div
      class="shadow-sm p-3"
    >
      <svg
        height="40"
        preserveAspectRatio="xMidYMid meet"
        version="1.1"
        viewBox="0 0 100% 40"
        width="100%"
      >
        <rect
          clip-path="url(#02wscj5har5e)"
          height="100%"
          style="fill: url(#nk01vtv2vem);"
          width="100%"
          x="0"
          y="0"
        />
        <defs>
          <clippath
            id="02wscj5har5e"
          >
            
            <circle
              cx="15"
              cy="20"
              r="15"
            />
            <rect
              height="40"
              width="100%"
              x="45"
              y="5"
            />
            
          </clippath>
          <lineargradient
            id="nk01vtv2vem"
          >
            <stop
              offset="0%"
              stop-color="#f3f3f3"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-2; 1"
              />
            </stop>
            <stop
              offset="50%"
              stop-color="#ecebeb"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-1.5; 1.5"
              />
            </stop>
            <stop
              offset="100%"
              stop-color="#f3f3f3"
              stop-opacity="1"
            >
              <animate
                attributeName="offset"
                dur="2s"
                repeatCount="indefinite"
                values="-1; 2"
              />
            </stop>
          </lineargradient>
        </defs>
      </svg>
    </div>
    
  </div>
</div>
`;

exports[`Storyshots Tasks/PureTaskList Loading 1`] = `
<div
  style="margin: 3em;"
>
  <div
    class="shadow-md w-100"
  >
    
    <div
      class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3 hover:bg-blue-200"
    >
      <div
        class="flex flex-row items-center space-x-2"
      >
        <label
          class="inline-flex items-center"
          for="task-6-checked"
        >
          <input
            class="border-cyan-600 form-checkbox border rounded text-indigo-600"
            disabled=""
            id="task-6-checked"
            name="checked"
            type="checkbox"
          />
           
        </label>
        <label
          class="inline-flex items-center"
          for="task-6-title"
        >
          <input
            class="bg-inherit form-input"
            id="task-6-title"
            placeholder="Input title"
            readonly=""
          />
           
        </label>
      </div>
      <button>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-star fa-w-18 text-yellow-500"
          data-icon="star"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class=""
            d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    <div
      class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3 hover:bg-blue-200"
    >
      <div
        class="flex flex-row items-center space-x-2"
      >
        <label
          class="inline-flex items-center"
          for="task-0-checked"
        >
          <input
            class="border-cyan-600 form-checkbox border rounded text-indigo-600"
            disabled=""
            id="task-0-checked"
            name="checked"
            type="checkbox"
          />
           
        </label>
        <label
          class="inline-flex items-center"
          for="task-0-title"
        >
          <input
            class="bg-inherit form-input"
            id="task-0-title"
            placeholder="Input title"
            readonly=""
          />
           
        </label>
      </div>
      <button>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-star fa-w-18 text-gray-200"
          data-icon="star"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class=""
            d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    <div
      class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3 hover:bg-blue-200"
    >
      <div
        class="flex flex-row items-center space-x-2"
      >
        <label
          class="inline-flex items-center"
          for="task-1-checked"
        >
          <input
            class="border-cyan-600 form-checkbox border rounded text-indigo-600"
            disabled=""
            id="task-1-checked"
            name="checked"
            type="checkbox"
          />
           
        </label>
        <label
          class="inline-flex items-center"
          for="task-1-title"
        >
          <input
            class="bg-inherit form-input"
            id="task-1-title"
            placeholder="Input title"
            readonly=""
          />
           
        </label>
      </div>
      <button>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-star fa-w-18 text-gray-200"
          data-icon="star"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class=""
            d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    <div
      class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3 hover:bg-blue-200"
    >
      <div
        class="flex flex-row items-center space-x-2"
      >
        <label
          class="inline-flex items-center"
          for="task-2-checked"
        >
          <input
            class="border-cyan-600 form-checkbox border rounded text-indigo-600"
            disabled=""
            id="task-2-checked"
            name="checked"
            type="checkbox"
          />
           
        </label>
        <label
          class="inline-flex items-center"
          for="task-2-title"
        >
          <input
            class="bg-inherit form-input"
            id="task-2-title"
            placeholder="Input title"
            readonly=""
          />
           
        </label>
      </div>
      <button>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-star fa-w-18 text-gray-200"
          data-icon="star"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class=""
            d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    <div
      class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3 hover:bg-blue-200"
    >
      <div
        class="flex flex-row items-center space-x-2"
      >
        <label
          class="inline-flex items-center"
          for="task-3-checked"
        >
          <input
            class="border-cyan-600 form-checkbox border rounded text-indigo-600"
            disabled=""
            id="task-3-checked"
            name="checked"
            type="checkbox"
          />
           
        </label>
        <label
          class="inline-flex items-center"
          for="task-3-title"
        >
          <input
            class="bg-inherit form-input"
            id="task-3-title"
            placeholder="Input title"
            readonly=""
          />
           
        </label>
      </div>
      <button>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-star fa-w-18 text-gray-200"
          data-icon="star"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class=""
            d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    <div
      class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3 hover:bg-blue-200"
    >
      <div
        class="flex flex-row items-center space-x-2"
      >
        <label
          class="inline-flex items-center"
          for="task-4-checked"
        >
          <input
            class="border-cyan-600 form-checkbox border rounded text-indigo-600"
            disabled=""
            id="task-4-checked"
            name="checked"
            type="checkbox"
          />
           
        </label>
        <label
          class="inline-flex items-center"
          for="task-4-title"
        >
          <input
            class="bg-inherit form-input"
            id="task-4-title"
            placeholder="Input title"
            readonly=""
          />
           
        </label>
      </div>
      <button>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-star fa-w-18 text-gray-200"
          data-icon="star"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class=""
            d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    
  </div>
</div>
`;

exports[`Storyshots Tasks/PureTaskList With Pinned Tasks 1`] = `
<div
  style="margin: 3em;"
>
  <div
    class="shadow-md w-100"
  >
    
    <div
      class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3 hover:bg-blue-200"
    >
      <div
        class="flex flex-row items-center space-x-2"
      >
        <label
          class="inline-flex items-center"
          for="task-0-checked"
        >
          <input
            class="border-cyan-600 form-checkbox border rounded text-indigo-600"
            disabled=""
            id="task-0-checked"
            name="checked"
            type="checkbox"
          />
           
        </label>
        <label
          class="inline-flex items-center"
          for="task-0-title"
        >
          <input
            class="bg-inherit form-input"
            id="task-0-title"
            placeholder="Input title"
            readonly=""
          />
           
        </label>
      </div>
      <button>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-star fa-w-18 text-gray-200"
          data-icon="star"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class=""
            d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    <div
      class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3 hover:bg-blue-200"
    >
      <div
        class="flex flex-row items-center space-x-2"
      >
        <label
          class="inline-flex items-center"
          for="task-1-checked"
        >
          <input
            class="border-cyan-600 form-checkbox border rounded text-indigo-600"
            disabled=""
            id="task-1-checked"
            name="checked"
            type="checkbox"
          />
           
        </label>
        <label
          class="inline-flex items-center"
          for="task-1-title"
        >
          <input
            class="bg-inherit form-input"
            id="task-1-title"
            placeholder="Input title"
            readonly=""
          />
           
        </label>
      </div>
      <button>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-star fa-w-18 text-gray-200"
          data-icon="star"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class=""
            d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    <div
      class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3 hover:bg-blue-200"
    >
      <div
        class="flex flex-row items-center space-x-2"
      >
        <label
          class="inline-flex items-center"
          for="task-2-checked"
        >
          <input
            class="border-cyan-600 form-checkbox border rounded text-indigo-600"
            disabled=""
            id="task-2-checked"
            name="checked"
            type="checkbox"
          />
           
        </label>
        <label
          class="inline-flex items-center"
          for="task-2-title"
        >
          <input
            class="bg-inherit form-input"
            id="task-2-title"
            placeholder="Input title"
            readonly=""
          />
           
        </label>
      </div>
      <button>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-star fa-w-18 text-gray-200"
          data-icon="star"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class=""
            d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    <div
      class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3 hover:bg-blue-200"
    >
      <div
        class="flex flex-row items-center space-x-2"
      >
        <label
          class="inline-flex items-center"
          for="task-3-checked"
        >
          <input
            class="border-cyan-600 form-checkbox border rounded text-indigo-600"
            disabled=""
            id="task-3-checked"
            name="checked"
            type="checkbox"
          />
           
        </label>
        <label
          class="inline-flex items-center"
          for="task-3-title"
        >
          <input
            class="bg-inherit form-input"
            id="task-3-title"
            placeholder="Input title"
            readonly=""
          />
           
        </label>
      </div>
      <button>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-star fa-w-18 text-gray-200"
          data-icon="star"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class=""
            d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    <div
      class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3 hover:bg-blue-200"
    >
      <div
        class="flex flex-row items-center space-x-2"
      >
        <label
          class="inline-flex items-center"
          for="task-4-checked"
        >
          <input
            class="border-cyan-600 form-checkbox border rounded text-indigo-600"
            disabled=""
            id="task-4-checked"
            name="checked"
            type="checkbox"
          />
           
        </label>
        <label
          class="inline-flex items-center"
          for="task-4-title"
        >
          <input
            class="bg-inherit form-input"
            id="task-4-title"
            placeholder="Input title"
            readonly=""
          />
           
        </label>
      </div>
      <button>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-star fa-w-18 text-gray-200"
          data-icon="star"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class=""
            d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    <div
      class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3 hover:bg-blue-200"
    >
      <div
        class="flex flex-row items-center space-x-2"
      >
        <label
          class="inline-flex items-center"
          for="task-5-checked"
        >
          <input
            class="border-cyan-600 form-checkbox border rounded text-indigo-600"
            disabled=""
            id="task-5-checked"
            name="checked"
            type="checkbox"
          />
           
        </label>
        <label
          class="inline-flex items-center"
          for="task-5-title"
        >
          <input
            class="bg-inherit form-input"
            id="task-5-title"
            placeholder="Input title"
            readonly=""
          />
           
        </label>
      </div>
      <button>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-star fa-w-18 text-gray-200"
          data-icon="star"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 576 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class=""
            d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    
  </div>
</div>
`;

exports[`Storyshots Tasks/TaskItem Archived 1`] = `
<div
  class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3"
>
  <div
    class="flex flex-row items-center space-x-2"
  >
    <label
      class="inline-flex items-center"
      for="task-1-checked"
    >
      <input
        class="border-cyan-600 form-checkbox border rounded text-indigo-600"
        disabled=""
        id="task-1-checked"
        name="checked"
        type="checkbox"
      />
       
    </label>
    <label
      class="inline-flex items-center"
      for="task-1-title"
    >
      <input
        class="bg-inherit form-input"
        id="task-1-title"
        placeholder="Input title"
        readonly=""
      />
       
    </label>
  </div>
  <button>
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-star fa-w-18 text-yellow-500"
      data-icon="star"
      data-prefix="fas"
      focusable="false"
      role="img"
      viewBox="0 0 576 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        class=""
        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
        fill="currentColor"
      />
    </svg>
  </button>
</div>
`;

exports[`Storyshots Tasks/TaskItem Default 1`] = `
<div
  style="margin: 3em;"
>
  <div
    class="shadow-md w-100"
  >
    <div
      class="flex flex-col space-y-3 justify-center items-center p-5"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-check fa-w-16 fa-2x text-green-600"
        data-icon="check"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          class=""
          d="M173.898 439.404l-166.4-166.4c-9.997-9.997-9.997-26.206 0-36.204l36.203-36.204c9.997-9.998 26.207-9.998 36.204 0L192 312.69 432.095 72.596c9.997-9.997 26.207-9.997 36.204 0l36.203 36.204c9.997 9.997 9.997 26.206 0 36.204l-294.4 294.401c-9.998 9.997-26.207 9.997-36.204-.001z"
          fill="currentColor"
        />
      </svg>
      <p
        class="font-bold"
      >
        You have no tasks
      </p>
      <p
        class=""
      >
        Sit back and relax
      </p>
    </div>
  </div>
</div>
`;

exports[`Storyshots Tasks/TaskItem Pinned 1`] = `
<div
  class="flex flex-row items-center space-x-4 w-100 justify-between shadow-sm p-3"
>
  <div
    class="flex flex-row items-center space-x-2"
  >
    <label
      class="inline-flex items-center"
      for="task-1-checked"
    >
      <input
        class="border-cyan-600 form-checkbox border rounded text-indigo-600"
        disabled=""
        id="task-1-checked"
        name="checked"
        type="checkbox"
      />
       
    </label>
    <label
      class="inline-flex items-center"
      for="task-1-title"
    >
      <input
        class="bg-inherit form-input"
        id="task-1-title"
        placeholder="Input title"
        readonly=""
      />
       
    </label>
  </div>
  <button>
    <svg
      aria-hidden="true"
      class="svg-inline--fa fa-star fa-w-18 text-gray-200"
      data-icon="star"
      data-prefix="fas"
      focusable="false"
      role="img"
      viewBox="0 0 576 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        class=""
        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
        fill="currentColor"
      />
    </svg>
  </button>
</div>
`;
